src/sentry_logger.cpp

```diff
@@ -152,11 +134,27 @@ void SentryLogger::_log_error(const char *p_func, const char *p_file, int p_line
 		}
 
 		SentrySDK::get_singleton()->get_internal_sdk()->capture_error(
-				error_types[p_error_type],
+				error_types[int(p_error_type)],
 				p_rationale,
-				godot_error_to_sentry_level(p_error_type),
+				sentry::get_sentry_level_for_godot_error_type(p_error_type),
 				{ stack_frame });
 	}
+
+	// Capture error as breadcrumb.
+	if (as_breadcrumb) {
+		Dictionary data;
+		data["function"] = String(p_func);
+		data["file"] = String(p_file);
+		data["line"] = p_line;
+		data["godot_error_type"] = String(error_types[int(p_error_type)]);
+
+		SentrySDK::get_singleton()->add_breadcrumb(
```

Comment:
Based on the screenshot loks like the order is inverted (breadcrumb added before the error) but here it seems the breadcrumb is added after.

Do we first capture the error, and then add the breadcrumb so the next error contains a breadcrumb to the previous error?

Reply:
> Do we first capture the error, and then add the breadcrumb so the next error contains a breadcrumb to the previous error?

Yes, as in code. I can swap order in options, if it's confusing.

Reply:
Done